### DISCUSSION TITLE
[Feature] In the iOS app, the metadata of the backed up file is different.

### DISCUSSION THREAD

**Discussion by paijjang** [immich-app/immich#5712] [Feature Request]
### The feature

It was written with a translator, so the language is awkward. I apologize in advance.
---------------------------------------------

I am currently using immich as Docker on Synology. I think immich is the best photo backup server and program I've ever used.
However, i have one problem.
I don't know if this is a bug or not, so I'm leaving it as a discussion section.
(In this case, the same symptoms appear in synology photos app backup and immich app backup, which I have currently tested)
In the case of portrait photos or cinematic videos backed up with immich ios app, delete the original video from the iPhone, re-download it as iOS Photos from immich, and run it.
Then, the menu for turning portrait mode on and off, which was possible in the original version, will not work.
Cinematic video is also only capable of HDR, and the unique functions of cinematics (focus change, etc.) do not work.
However, after backing up with an app called photosync (third party), adding it to the immich server, downloading it through immich and running it in iOS Photos, editing that was possible in the original (turning on or off portrait mode, editing cinematic video) was possible. .
So, when comparing the actual exif metadata, something was confirmed to be different.
(In the photosync file, there is an additional item called Region)
I would like to ask if immich can also include this information when backing up photos. If this information is added, I think it will be more convenient to use immich.

Differences in exif metadata (syntax that only exists in files backed up with photosync)

Region Area Y                   : 0.36199999999999999
Region Area W                   : 0.11299999999999999
Region Area X                   : 0.42749999999999994
Region Area H                   : 0.15799999999999997
Region Area Unit                : normalized
Region Type                     : Focus
Region Extensions               : 
Region Applied To Dimensions H  : 3024
Region Applied To Dimensions W  : 4032
Region Applied To Dimensions Unit: pixel

[IMG_6629.HEIC(backup_for_immich).TXT](https://github.com/immich-app/immich/files/13681611/IMG_6629.HEIC.backup_for_immich.TXT)
[IMG_6629.HEIC(backup_for_photosync).TXT](https://github.com/immich-app/immich/files/13681612/IMG_6629.HEIC.backup_for_photosync.TXT)

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by moritzbindewald** [comment]
Pls report as issue or maybe @alextran1502 can move it? Thx!

**Reply by paijjang** [reply]
yes. thx to ur advise
i’ll move to issue tab
